tests/php/src/WithoutBlockPreRendering.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Trait WithoutBlockPreRendering.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Tests;
+
+/**
+ * Helper trait to skip block pre-rendering for tests that are not block related.
+ *
+ * @package AmpProject\AmpWP
+ */
+trait WithoutBlockPreRendering {
+
+	/**
+	 * Set up.
+	 */
+	public function setUp() { //phpcs:ignore WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid
+		parent::setUp();
+
+		add_filter(
+			'render_block_data',
+			static function( $parsed_block ) {
+				global $post;
+
+				if (
+				null === $post &&
+				defined( 'GUTENBERG_VERSION' ) &&
+				version_compare( '8.1.0', GUTENBERG_VERSION, '<=' )
+				) {
```

Comment:
Indentation is off here:
```suggestion
				if (
					null === $post &&
					defined( 'GUTENBERG_VERSION' ) &&
					version_compare( '8.1.0', GUTENBERG_VERSION, '<=' )
				) {
```

This also lets me assume that PHPCS is not reaching this part of the code, so its config needs to be examined.

Reply:
The rules [`PSR12.ControlStructures.BooleanOperatorPlacement`](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties#psr12controlstructuresbooleanoperatorplacement) and [`PSR12.ControlStructures.ControlStructureSpacing`](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties#psr12controlstructurescontrolstructurespacing) should do the job here. I'll include them in a separate PR though, as they report a lot of errors in files not related to these changes.

Reply:
But this file is not PSR-12, it's WPCS, and that should catch the above I think.

Reply:
I'm not familiar with a rule for spacing of conditions over multiple lines in WPCS, but there is one for [spacing of parameters](https://github.com/WordPress/WordPress-Coding-Standards/blob/e0db33d7b37c0f945539ed17ee5d581326fd1ab0/WordPress-Core/ruleset.xml#L246) for functions calls.

Reply:
I might misremember, but I thought WPCS included a rule for enforcing 1 tab of indentation when you change a control structure from single-line to multi-line...